src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -15,29 +15,21 @@ class NUnit extends ConventionTask {
     def nunitVersion
     def nunitDownloadUrl
     List testAssemblies
-    def include
-    def exclude
-    def where
+
     def framework
     def verbosity
     def config
     def timeout
 
-    // TODO: shouldn't we just make it a single variable and detect which case it is when invoking NUnit instead?
-    // runList and run are mutually exclusive
-    def runList
-    def run
-
-    // testList and test are mutually exclusive, NUnit3-specific
-    def testList
-    def test
-
     def reportFolder
     boolean useX86 = false
     boolean shadowCopy = false
     String reportFileName = 'TestResult.xml'
     boolean ignoreFailures = false
-    boolean parallel_forks = true
+    boolean parallelForks = true
+
+    def test
```

Comment:
If we broke some stuff already, how about we use a single variable, which is always a list? That will include the case when there's a single item.